gps/gps.cpp

```diff
@@ -150,7 +150,8 @@ string GPS::usbnmeaprobe(vector<string> devices) {
             }
         }
     }
-    return NULL;
+    cout << "No NMEA device found\n";
```

Comment:
Does this get printed in an infinite loop or only once?

Reply:
Only when there exist ttyUSB* and non of the devices respond then it would print out every failed retry, which is around 5 seconds

Reply:
Ok that's reasonable